modules/presence/publish.c

```diff
@@ -616,6 +619,9 @@ int handle_publish(struct sip_msg* msg, str* sender_uri)
 		goto error;
 	}
 
+	/* send event E_PRESENCE_PUBLISH */
+	presence_raise_event(presence_event_id, &presentity);
+
```

Comment:
@hafkensite , is the event raising moved after the update_presentity() related to obtaining the old_etag info ?